### ISSUE TITLE
ML: Deep Model #1 (24 Layers): Configuration#5: Training model and store the results in the directory for 30 and 50 epochs

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#269]
Model Architect: 

- 4Conv(Conv+Conv+MaxPool+Dropout)
- 4Dense(Dense+Dropout)

1. Put the training on the Grid by using jetscape-ml-tensorflow-cnn.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
30 epochs

Epoch 30/30
839936/840000 [============================>.] - ETA: 0s - loss: 0.9741 - accuracy: 0.9239  
Epoch 00030: val_accuracy did not improve from 0.92595
840000/84000
